source/includes/_scarts.md

```diff
@@ -0,0 +1,15 @@
+# SCA-RTS (PS21/19)
+
+## Overview
+In November 2021, the FCA set out new rules for standards on Strong Customer Authentication (SCA) in a change to the Regulatory Technical Standards on Strong Customer Authentication and Secure Communication (SCA-RTS).
```

Comment:
Can we reference the specific policy change here for readers who aren't already aware (ps21/19 I think)